content/variables/environment-variable-groups.md

```diff
@@ -31,9 +31,14 @@ Variables defined in environment variable groups work exactly as [Environment Va
 
 ## Storing sensitive values/files
 
-Entering values in the Variable value input and marking the **Secure** checkbox will automatically encrypt those values. However, note that in order to store files as secure environment variables, the file needs to be base64 encoded first. This can be done with the help of different OS-specific command lines. To use the file, you will have to decode it during the build. 
+Entering values in the Variable value input and marking the **Secure** checkbox will automatically encrypt those values. However, note that in order to store **_binary files_** as secure environment variables, first it needs to be **_base64 encoded_** locally. To use the files, you will have to decode them during the build.
 
-Files like keystore, provisioning profiles require to be encoded in base64.
+Some commonly known binary files that need to be base64 encoded. e.g.
+- Android keystore (.jks or .keystore)
+- Provisioning profiles when manual code signing (.mobileprovision)
+- iOS distribution certificate (.p12) when manual code signing.
+
+This can be done with the help of different OS-specific command lines':
```

Comment:
Maybe we can add another tip here about how to see if a file is a binary file or not using `less`?